### DISCUSSION TITLE
VenusOS (CerboGX or Raspberry) Bluetooth connection to the Neey Balancer directly over Bluetooth to NodeRed?

### DISCUSSION THREAD

**Discussion by moench13** [syssi/esphome-jk-bms#403] [Ideas]
Hi,

serialbattery uses the the bluetooth connection from the Raspberry or Cerbo GX on VenusOS do connected directly to the JK/BMS.
So my Question is... is it possible to use the bluetooth connection to conneced direkty to the Neey Balancer so i can use Cell Voltages or ON/OFF switch for a visual in NodeRed over VenusOS - so there is no esp32 needed?

Best Regard

**Comment by syssi** [comment]
I'm not aware of a project supporting the NEEY balancer using the Bluetooth stack of a Linux host. You could pick the  most important parts from this project to setup a BLE client connection on your own and implement the protocol.
